manim/animation/composition.py

```diff
@@ -81,6 +81,15 @@ def get_all_mobjects(self) -> Sequence[Mobject]:
         return list(self.group)
 
     def begin(self) -> None:
+        if self.run_time <= 0:
+            tmp = (
+                "please set a runtime > 0"
+                if len(self.animations) != 0
+                else "Please add at least one Animation"
+            )
+            raise ValueError(
+                f"{self} has a runtime of 0 seconds. Which cannot be rendered correctly! {tmp}."
```

Comment:
```suggestion
                f"{self} has a runtime of <= 0 seconds, which cannot be rendered correctly! please set a runtime > 0"
```

Reply:
Why exactly should we remove the differentiation between 0 runtime and no animations for the animation group ? They are two distinct errors.

Reply:
Oh, ignore that last part, copy-paste error 😅